e2e_tests_codefresh_pipeline.yaml

```diff
@@ -0,0 +1,51 @@
+version: "1.0"
+stages:
+  - "clone"
+  - "build"
+  - "e2e_test"
+steps:
+  main_clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "Labelbox/labelbox-python"
+    stage: "clone"
+    revision: "${{CF_BRANCH}}"
+  parallel_build:
+    type: "parallel"
+    stage: "build"
+    steps:
+      build_sdk_test:
+        description: "Building Docker image for sdk tests"
+        tag: "testing"
+        type: "build"
+        image_name: "sdk-tests"
+        disable_push: true
+        working_directory: "${{main_clone}}"
+        dockerfile: "Dockerfile"
+  e2e_test:
+    title: "Python SDK e2e tests"
+    description: "Running Python SDK e2e tests"
+    fail_fast: false
+    stage: "e2e_test"
+    image: "${{build_sdk_test}}"
+    working_directory: "IMAGE_WORK_DIR"
+    commands:
+      - pytest
+  sendSlack:
+    title: "Alert on failure"
```

Comment:
Maybe for a while we should alert on success too to start to make sure things are running smoothly